ortools/linear_solver/xpress_interface.cc

```diff
@@ -222,6 +222,9 @@ class XpressInterface : public MPSolverInterface {
     mRstat.clear();
     sync_status_ = MUST_RELOAD;
   }
+  // Adds a new feasible, infeasible or partial MIP solution for the problem to the Optimizer.
+  // The hint is read in the MPSolver where the user set it
+  virtual void SetHint();
```

Comment:
```suggestion
  void SetHint() override;
```
makes `virtual` implicit, ensures we're overriding an existing virtual method.

Reply:
we're actually not overriding the super's function, I shouldn't have set this to "virtual". Also I'll rename it to avoid confusion.